.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -28,63 +24,59 @@ body:
     id: actual
     attributes:
       label: Actual outcome
-      description: Please provide the output produced by the above code, which may be a screenshot, console output, etc. 
-      placeholder: if applicable, paste the console output and images here
+      description: The output produced by the above code; e.g. a screenshot, console output, etc.
+      placeholder: Paste the console output and images here
     validations:
       required: true
   - type: textarea
     id: expected
     attributes:
       label: Expected outcome
-      description: Please provide a description (or visual example) of the expected outcome from the code snippet. If this used to work in an earlier version of Matplotlib, please note the version it used to work on
+      description: Description (or visual example) of the expected outcome from the code snippet.
     validations:
       required: true
+  - type: textarea
+    id: details
+    attributes:
+      label: Detailed description
+      description: |
+        Context, boundary conditions, etc. Has this worked in earlier versions?
+        If possible: Root cause, suggested fix.
```

Comment:
maybe make this a more information box instead to differentiate from description? I'm not sure what you're trying to elicit here that's not covered by description...maybe a why do you think this bug is happening?
```yaml
label: More information
description: Any other information that may be helpful in understanding this bug
placeholder: |
* Has this worked before? anything different this time?
* What are the conditions under which this bug happens? input parameters, edge cases, etc?
* any theories about what's triggering this bug?
```
suggested fixes is another field

Reply:
Taken over your suggestions (with some minor rewording) but in the description not in the placeholder.